std/uuid.d

```diff
@@ -148,6 +148,12 @@ public struct UUID
                 179, 189, 251, 70])._toString() == "8ab3060e-2cba-4f23-b74c-b52db3bdfb46");
         }
 
+        // Reinterpret the UUID as an array of some other primitive.
+        @trusted ref T[16 / T.sizeof] asArrayOf(T)()
```

Comment:
`@return` was added once, but reverted soon. Currently compiler accepts only `return` and `@trusted`.
[Revert "accept @pure @nothrow @return attributes"](https://github.com/D-Programming-Language/dmd/pull/4349)
http://forum.dlang.org/thread/rtwbtxigfeupvykpbamh@forum.dlang.org?page=1